nncf/experimental/torch/nas/bootstrapNAS/search/evaluator.py

```diff
@@ -101,8 +101,7 @@ def add_to_cache(self, subnet_config_repr: Tuple[float, ...], measurement: float
         :param measurement: value for the evaluator's metric.
         :return:
         """
-        nncf_logger.info("Add to evaluator {name}: {subnet_config_repr}, {measurement}".format(
-                         name=self.name, subnet_config_repr=subnet_config_repr, measurement=measurement))
+        nncf_logger.info(f"Add to evaluator {self.name}: {subnet_config_repr}, {measurement}")
```

Comment:
Propose moving to debug
@jpablomch

Reply:
@ljaljushkin Moving to debug works for us. Thanks!

Reply:
Done